lib/screens/v2/widget_instance/departures_no_data.ex

```diff
@@ -4,11 +4,12 @@ defmodule Screens.V2.WidgetInstance.DeparturesNoData do
   alias Screens.Config.Screen
   alias Screens.Config.V2.Alerts
 
-  defstruct screen: nil, show_alternatives?: nil
+  defstruct screen: nil, show_alternatives?: nil, slot_names: []
```

Comment:
Why is this pluralized? In `departures.ex`, this widget is created with: 

```
   Enum.map(slot_ids, fn slot_id ->
      if Enum.all?(sections, &(&1.type == :no_data_section)) do
        %DeparturesNoData{screen: config, slot_names: [slot_id]}
      else ...
```

so `slot_id` should always be just one id.

Reply:
I did that mostly to leave room for an app to give slot options for the widget to be valid in. While that has made sense for previous widgets (evergreen is the main one that needs this), I'm not sure that's actually valuable for this one. So I think I'll change it to a single slot.